grid/project-management/projects/profile.supr

```diff
@@ -0,0 +1,40 @@
+"""
+Manage projects in project management systems.
+"""
+name = "project-management/projects"
+version = "1.0.0"
+
+"""
+Create a new task.
```

Comment:
Mind the description. Also, the first line is a title so it shouldn't be a sentence.

Since the use case doesn't have any input, it would be useful to mention how  the projects for listing are picked (are those all projects the user has permissions for?)

```suggestion
List Projects
```

Reply:
:facepalm: Copy & paste fail.

Agreed on the elaboration (though the use case will accept input in the future, alleviating the need for this a bit).